Translations/pt_BR.po

```diff
@@ -1607,181 +1601,157 @@ msgstr "Exibe chat."
 
 #: Source/diablo.cpp:1886
 msgid "Primary action"
-msgstr ""
+msgstr "Ação primária"
 
 #: Source/diablo.cpp:1887
 msgid "Attack monsters, talk to towners, lift and place inventory items."
 msgstr ""
+"Atacar monstros, conversar com os habitantes da cidade, tirar e colocar "
+"itens de inventário."
 
 #: Source/diablo.cpp:1901
-#, fuzzy
-#| msgid "Select Action"
 msgid "Secondary action"
-msgstr "Selecionar Ação"
+msgstr "Ação secundária"
 
 #: Source/diablo.cpp:1902
 msgid "Open chests, interact with doors, pick up items."
-msgstr ""
+msgstr "Abrir baús, interagir com portas, pegar itens."
 
 #: Source/diablo.cpp:1916
-#, fuzzy
-#| msgid "Select Action"
 msgid "Spell action"
-msgstr "Selecionar Ação"
+msgstr "Ação de feitiço"
 
 #: Source/diablo.cpp:1917
 msgid "Cast the active spell."
-msgstr ""
+msgstr "Lança o feitiço ativo."
 
 #: Source/diablo.cpp:1931
-#, fuzzy
-#| msgid "Cancel"
 msgid "Cancel action"
-msgstr "Cancelar"
+msgstr "Cancelar ação"
 
 #: Source/diablo.cpp:1932
-#, fuzzy
-#| msgid "Closes the game."
 msgid "Close menus."
-msgstr "Fechar o jogo."
+msgstr "Fechar os menus."
 
 #: Source/diablo.cpp:1957
 msgid "Move up"
-msgstr ""
+msgstr "Mover pra cima"
 
 #: Source/diablo.cpp:1958
-#, fuzzy
-#| msgid "Multi Player Characters"
 msgid "Moves the player character up."
-msgstr "Personagens Multijogador"
+msgstr "Move personagem para cima."
 
 #: Source/diablo.cpp:1963
 msgid "Move down"
-msgstr ""
+msgstr "Mover pra baixo"
 
 #: Source/diablo.cpp:1964
-#, fuzzy
-#| msgid "Multi Player Characters"
 msgid "Moves the player character down."
-msgstr "Personagens Multijogador"
+msgstr "Move personagem para baixo."
 
 #: Source/diablo.cpp:1969
-#, fuzzy
-#| msgid "$Movement:"
 msgid "Move left"
-msgstr "$Movimento:"
+msgstr "Mover pra esquerda"
 
 #: Source/diablo.cpp:1970
-#, fuzzy
-#| msgid "Multi Player Characters"
 msgid "Moves the player character left."
-msgstr "Personagens Multijogador"
+msgstr "Move personagem para esquerda."
 
 #: Source/diablo.cpp:1975
-#, fuzzy
-#| msgid "the night"
 msgid "Move right"
-msgstr "da noite"
+msgstr "Mover pra direita"
 
 #: Source/diablo.cpp:1976
-#, fuzzy
-#| msgid "Multi Player Characters"
 msgid "Moves the player character right."
-msgstr "Personagens Multijogador"
+msgstr "Move personagem para direita."
 
 #: Source/diablo.cpp:1981
 msgid "Stand ground"
-msgstr ""
+msgstr "Ficar parado"
```

Comment:
The direct translation doesn't work well here IMHO. Suggestion:
"Manter-se firme"